src/List/test/ListSpec.tsx

```diff
@@ -1,59 +1,60 @@
 import React from 'react';
-import { render, waitFor } from '@testing-library/react';
+import { render, waitFor, screen, cleanup } from '@testing-library/react';
 import ReactTestUtils from 'react-dom/test-utils';
-import { getDOMNode } from '@test/testUtils';
 import { testStandardProps } from '@test/commonCases';
 import List from '../List';
 import Sinon from 'sinon';
 
 describe('List', () => {
+  afterEach(cleanup);
+
   testStandardProps(<List />);
 
   it('Should render a List', () => {
-    const domNode = getDOMNode(<List />);
-    assert.include(domNode.className, 'rs-list');
+    render(<List data-testid="list" />);
+    expect(screen.getByTestId('list')).to.have.class('rs-list');
```

Comment:
`getByRole` should be chosen first instead of `getByTestId`.

```
screen.getByRole('list')
```
> https://testing-library.com/docs/queries/about#priority